.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: Python CI
 
 on:
   push:
-    branches: [main]
+    branches: [master]
```

Comment:
You also have `main` instead of `master` in Readme file. please make sure to update that and all other instances as well.